src/flamenco/types/fd_types_custom.c

```diff
@@ -172,17 +172,15 @@ void fd_solana_vote_account_decode_unsafe( fd_solana_vote_account_t * self, fd_b
   ulong data_len;
   fd_bincode_uint64_decode_unsafe( &data_len, ctx );
   if( data_len ) {
-    uchar * data = fd_valloc_malloc( ctx->valloc, 8UL, data_len );
-    fd_bincode_bytes_decode_unsafe( data, data_len, ctx );
-
     FD_SCRATCH_SCOPE_BEGIN {
       /* Deserialize content */
       fd_vote_block_timestamp_t vote_ts;
       fd_vote_state_versioned_t vs[1];
       fd_bincode_decode_ctx_t decode =
-          { .data    = data,
-            .dataend = data + data_len,
-            .valloc  = fd_scratch_virtual() };
+        { .data    = (uchar *)ctx->data,
```

Comment:
We cannot decode into the ctx here, this method is designed to save only the fields we need and keeping around the entire variable length data buffer